kairos-data/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+use deadpool_diesel::postgres::{Runtime, Manager};
+use diesel::{prelude::*, select, sql_types::Text};
+use tracing::{debug, info};
+
+#[cfg(feature="migrations")]
+use diesel_migrations::{embed_migrations, EmbeddedMigrations, MigrationHarness};
+
+pub use deadpool_diesel::postgres::Pool;
+pub use diesel::{insert_into, prelude};
+
+pub mod errors;
+pub mod schema;
+pub mod transaction;
+
+#[cfg(feature="migrations")]
+pub const MIGRATIONS: EmbeddedMigrations = embed_migrations!();
```

Comment:
I would prefer the migrations directory to be parametrzed and configurable when I call `run`. It could also be an option and default to whatever the default in the `Cargo.toml` is.

Reply:
The migrations directory can be configured from diesel.toml